packages/client/src/components/manage-users/pay-off-user-dialog.vue

```diff
@@ -382,16 +384,35 @@ const {
   retailLocationId: selectedLocation.value.id,
 }));
 
-const {
-  soldBookCopies: soldCopies,
-  loading: soldLoading,
-  // refetch: refetchSoldBookCopies,
-} = useGetSoldBookCopiesQuery(() => ({
-  userId: props.user.id,
-  retailLocationId: selectedLocation.value.id,
-}));
-const totalCheckoutMoney = ref(0);
-const totalCheckedOutMoney = ref(0);
+const { soldBookCopies: soldCopies, loading: soldLoading } =
+  useGetSoldBookCopiesQuery(() => ({
+    userId: props.user.id,
+    retailLocationId: selectedLocation.value.id,
+  }));
+
+const getSettledOrToSettleCopiesPriceSum = (
+  settledAt: number | null | undefined,
+  book: BookSummaryFragment,
+  settled: boolean,
+) =>
+  (settled ? settledAt : !settledAt)
+    ? (book.originalPrice *
+        (props.user.discount
+          ? selectedLocation.value.buyRate
+          : selectedLocation.value.sellRate)) /
+      100
```

Comment:
When giving money back to the user who gave us the books, we will always have to deal in terms of buyRate, regardless of what happened to that book sale.
Even if the book was refunded (but not settled yet) it needs to be counted among the money the user will get back from the settlement operation.

Remember that here the protagonist is Mercatino: and back to users it will always give only the `buyRate` money.

Reply:
In general make sure to check this calculation, it wrong for some reason here. The money to pay off is calculated like this:
all_sold_books_not_settled_yet_original_price * buyRate
+
all_refunded_book_not_settled_yet_original_price * buyRate